Client/Frontend/Browser/Playlist/PlaylistToast.swift

```diff
@@ -146,6 +146,16 @@ class PlaylistToast: Toast {
             $0.contentEdgeInsets = UIEdgeInsets(top: 10.0, left: 10.0, bottom: 10.0, right: 20.0)
             $0.titleEdgeInsets = UIEdgeInsets(top: 0.0, left: 10.0, bottom: 0.0, right: -10.0)
             
+            let colors = [#colorLiteral(red: 0.4352941176, green: 0.2980392157, blue: 0.8235294118, alpha: 1), #colorLiteral(red: 0.7490196078, green: 0.07843137255, blue: 0.6352941176, alpha: 1), #colorLiteral(red: 0.968627451, green: 0.2274509804, blue: 0.1098039216, alpha: 1)]
+            let gradientView = GradientView(colors: colors,
+                                            positions: [0.1581, 0.6317, 1.0],
+                                            angle: 304.74)
```

Comment:
Can you export this value to new file called `Gradient.swift` in `BraveUI` framework?
This may require moving `GradientView` there too

The gradient should be named `gradient02-alt-light`  or something similar

Reply:
Fixed.